### PR TITLE
Feat add method field in RequestBuilder

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: src/test/java/no/unit/nva/testutils/HandlerRequestBuilderTest.java [hunk 1]
```diff
@@ -20,6 +21,8 @@
     public static final String PATH_PARAMETERS = "pathParameters";
     public static final String QUERY_PARAMETERS = "queryStringParameters";
     public static final String REQUEST_CONTEXT = "requestContext";
+    public static final String SOME_METHOD = "POST";
+    private static final String METHOD = "httpMethod";
```
Comment (brinxmat) [inline src/test/java/no/unit/nva/testutils/HandlerRequestBuilderTest.java]: "method" as this is what is reflected in the code

Comment (brinxmat) [reply src/test/java/no/unit/nva/testutils/HandlerRequestBuilderTest.java]: I fixed this in PR https://github.com/BIBSYSDEV/nva-testutils/pull/12, you will want to add travis CI, codacy to this project as the tests did not pass for the PR

**Review by brinxmat — CHANGES_REQUESTED** [review]

**Review by janbj — DISMISSED** [review]

File: build.gradle [hunk 2]
```diff
@@ -31,3 +35,64 @@ test {
...
+jacocoTestReport {
+    reports {
+        xml.enabled true
+    }
+}
+
+pmd {
+    ruleSetConfig = rootProject.resources.text.fromFile('config/pmd/ruleset.xml')
+    ruleSets = []
+    ignoreFailures = true
```
Comment (axthosarouris) [inline build.gradle:60]: allow build to pass. Bring library up to standards on next PR.

File: build.gradle [hunk 3]
```diff
@@ -31,3 +35,64 @@ test {
...
+check.dependsOn jacocoTestCoverageVerification
+jacocoTestCoverageVerification.dependsOn(jacocoTestReport)
+
+jacocoTestCoverageVerification {
+    violationRules {
+        rule {
+            limit {
+                counter = 'METHOD'
+                value = 'COVEREDRATIO'
+                minimum = 0.5
```
Comment (axthosarouris) [inline build.gradle:86]: allow PR to succeed. Bring library up to standards on next PR.

**Review by janbj — APPROVED** [review]

**Review by brinxmat — APPROVED** [review]

**Pull request merged**